src/stylesheets/components/_graphic-list.scss

```diff
@@ -1,18 +1,20 @@
 // A collection of media block objects
 
 .usa-graphic_list {
+  @include typeset;
+
   .usa-graphic_list-row {
     .usa-media_block {
-      margin-bottom: $site-margins;
+      margin-bottom: units($theme-site-margins);
 
-      @include media($medium-screen) {
-        margin-bottom: $site-margins * 2;
+      @include at-media(tablet) {
+        margin-bottom: units($theme-site-margins) * 2;
       }
     }
 
     &:last-child {
       .usa-media_block {
-        @include media($medium-screen) {
+        @include at-media(tablet) {
```

Comment:
Quote the strings in functions and mixins: `@include at-media('tablet')`
Though the fns and mixins _will_ work without them...

Reply:
That's interesting, I was following the documentation where they are not quoted: https://v2.designsystem.digital.gov/utilities/layout-grid/#example-usage also I have not been quoting any media queries in any components by following this example.

Reply:
Oops! Docs is wrong :(

Reply:
Updated to unit values in 1688ff6, but we will also need to update that file.

Reply:
Updated to use quoted text in 185b7d8, but we will also need to go through all the components and update... not to mention updating the docs 😉